codegen/server/local.ts

```diff
@@ -0,0 +1,58 @@
+import { PermanentMemo } from "../../util/memo.ts"
+import { Cache, FsCache } from "./cache.ts"
+import { CapiCodegenServer } from "./capi_repo.ts"
+import { getModuleIndex } from "./git_utils.ts"
+
+export class LocalCapiCodegenServer extends CapiCodegenServer {
+  version
+  cache: Cache = new FsCache("target/codegen", this.abortController.signal)
```

Comment:
Why the explicit `Cache` typing? Removing doesn't appear to adversely impact star's typecheck.

Reply:
At one point in time I had a test where I set it to be a different cache (`InMemoryCache`, pretty sure). I don't anymore, but future tests might make use of that, so I'm inclined to leave it.

Reply:
Actually, `codecs.test.ts` should be using `InMemoryCache` right now. I'll fix that.